vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputProps.h

```diff
@@ -118,6 +118,7 @@ class WindowsTextInputProps final : public ViewProps, public BaseTextProps {
   bool clearTextOnSubmit{false};
   std::vector<CompWindowsTextInputSubmitKeyEventsStruct> submitKeyEvents{};
   bool autoFocus{false};
+  std::string textAlign{};
```

Comment:
we should use facebook::react::TextAlignment here, rather than a string

Reply:
Done!